packages/vite/src/node/plugins/metadata.ts

```diff
@@ -0,0 +1,23 @@
+import type { Plugin } from '../plugin'
+
+export interface ChunkMetadata {
```

Comment:
This should be exported from `src/node/index.ts` so plugin devs can extend it with interface merging.